kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/facade/CompanyFacade.java

```diff
@@ -0,0 +1,66 @@
+package com.kodilla.hibernate.manytomany.facade;
+
+import com.kodilla.hibernate.manytomany.Company;
+import com.kodilla.hibernate.manytomany.Employee;
+import com.kodilla.hibernate.manytomany.dao.CompanyDao;
+import com.kodilla.hibernate.manytomany.dao.EmployeeDao;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CompanyFacade {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CompanyFacade.class);
+
+    public List<Company> listOfCompanies(CompanyDao companyDao, String threeChars)
```

Comment:
1. `CompanyDao` nie powinno być argumentem metody `listOfCompanies` (podobnie `EmployeeDao` w metodzie `listEmployees`. To są zależności komponentu `CompanyFacade`, powinny być wstrzyknięte do pól beana (najlepiej przez konstruktor).
2. Nazwy metod - jeśli coś wyszukują, najlepiej użyć prefiksu `find` na wzór różnych klas standardowych Javy.